src/client/Microsoft.Identity.Client/Instance/Discovery/NetworkMetadataProvider.cs

```diff
@@ -101,8 +101,11 @@ private Uri ComputeHttpEndpoint(Uri authority, RequestContext requestContext)
                 authority.Host :
                 AadAuthority.DefaultTrustedHost;
 
-            string instanceDiscoveryEndpoint =  UriBuilderExtensions.GetHttpsUriWithOptionalPort(
-                $"https://{discoveryHost}/common/discovery/instance",
+            string instanceDiscoveryEndpoint = UriBuilderExtensions.GetHttpsUriWithOptionalPort(
+                string.Format(
```

Comment:
I reccomend keeping the string interpolation because it is generally faster and more efficient.
Invariant culture shouldnt be needed for uri's right?